modules/client-configuration/pages/cobbler.adoc

```diff
@@ -82,10 +88,17 @@ This example:
 * Creates a class called ``PXE``.
 * Identifies systems as ``PXEClient`` if they are configured with PXE as the first boot priority.
 * Directs PXEClients to the Cobbler server at ``192.168.2.1``.
-* Retrieves the [path]``pxelinux.0``bootloader file.
+* Retrieves the [path]``grub/grub.efi`` bootloader file for EFI PXE clients.
+* Retrieves the [path]``pxelinux.0`` bootloader file for legacy BIOS PXE clients.
 +
 . Save the file.
 
+.Procedure: Example for configuring ISC DHCP Server using DHCPd formula
+. Configure DHCPd formula as described in xref:salt:formula-dhcpd.adoc[]
+. Use [path]``grub/grub.efi`` in [guimenu]``Filename EFI`` field for EFI PXE support
+. Use [path]``pxelinux.0`` in [guimenu]``Filename`` field for legacy BIOS support
```

Comment:
```suggestion
. In the [guimenu]``Filename EFI`` field, for EFI PXE support type [path]``grub/grub.efi``.
  Alternatively, for legacy BIOS support type [path]``pxelinux.0``.
```

Reply:
This is not exactly what I meant. I mean that user should put both values there to support both legacy and efi PXE. So that this example will result in same behaviour as the manual example above.

Reply:
Ah, I understand, let's go back to two steps then, assuming they are two different fields. How about this:

```suggestion
. In the  [guimenu]``Filename EFI`` field, type [path]``grub/grub.efi`` to enable EFI PXE support.
. In the  [guimenu]``Filename`` field, type [path]``pxelinux.0`` to enable legacy BIOS support.
```